src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
```

Comment:
Ensure that the `.logo` element exists in the DOM when the script runs. If it is dynamically added, consider using a DOMContentLoaded event listener or similar to ensure the element is available.